pkgs/applications/editors/jetbrains/bin/versions.json

```diff
@@ -3,34 +3,34 @@
     "aqua": {
       "update-channel": "Aqua EAP licensing:EAP",
       "url-template": "https://download.jetbrains.com/aqua/aqua-{version}.tar.gz",
-      "version": "2024.1 Public Preview",
-      "sha256": "027cac4bfab747ef97f055b63eff8a69cefc32751fcca9510c3eccaf7c36b2be",
-      "url": "https://download.jetbrains.com/aqua/aqua-241.15989.28.tar.gz",
-      "build_number": "241.15989.28"
+      "version": "2023.3 Public Preview",
+      "sha256": "7fbc5565208acd0f4428c10823778c0d2bfd192e4cfce74ee2788477dacad5e0",
+      "url": "https://download.jetbrains.com/aqua/aqua-233.14475.33.tar.gz",
+      "build_number": "233.14475.33"
```

Comment:
Looks like the aqua is being downgraded for some reason. Furthermore latest build should be 242.20224.331. I think we need to switch to the Release channel now, at least looking at https://www.jetbrains.com/updates/updates.xml it looks like a "RELEASE" channel is now available and not just public preview.

Reply:
Thanks for catching that. I believe I've changed it to the `RELEASE` channel and re-run update script.